selfdrive/modeld/thneed/thneed

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+export LD_LIBRARY_PATH="/data/pythonpath/phonelibs/snpe/aarch64/:$HOME/openpilot/phonelibs/snpe/larch64:$HOME/openpilot/phonelibs/snpe/x86_64-linux-clang:$LD_LIBRARY_PATH"
```

Comment:
openpilot is now always in `/data/openpilot`

Reply:
oh this file isn't used, let me move to debug and fix